.buildkite/scripts/steps/code_coverage/ingest.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+is_test_execution_step
+
+export CODE_COVERAGE=1
+echo "--- Reading Kibana stats cluster creds from vault"
+export USER_FROM_VAULT="$(retry 5 5 vault read -field=username secret/kibana-issues/prod/coverage/elasticsearch)"
+export PASS_FROM_VAULT="$(retry 5 5 vault read -field=password secret/kibana-issues/prod/coverage/elasticsearch)"
+export HOST_FROM_VAULT="$(retry 5 5 vault read -field=host secret/kibana-issues/prod/coverage/elasticsearch)"
+export TIME_STAMP=$(date +"%Y-%m-%dT%H:%M:00Z")
+
+echo "--- Download previous git sha"
+.buildkite/scripts/steps/code_coverage/reporting/downloadPrevSha.sh
+previousSha=$(cat downloaded_previous.txt)
+
+echo "--- Upload new git sha"
+.buildkite/scripts/steps/code_coverage/reporting/uploadPrevSha.sh
+
+.buildkite/scripts/bootstrap.sh
+ 
+echo "--- Download coverage arctifacts"
+buildkite-agent artifact download target/kibana-coverage/jest/* . --build "${KIBANA_BUILD_ID:-$BUILDKITE_BUILD_ID}"
```

Comment:
You should remove `--build "${KIBANA_BUILD_ID:-$BUILDKITE_BUILD_ID}"` from both of these. Removing it will mean that you always get the artifacts from the currently running build